### PR TITLE
jwtBuilder configurable nbf claim

### PR DESCRIPTION
**Pull request by mmerali96**
Implements #11868

### CODE REVIEW

File: dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -90,3 +90,5 @@ jwkEndpointUrl.desc=Specifies a JWK end point URL.
 useSystemPropertiesForHttpClientConnections=Use system properties for HTTP client connections
 useSystemPropertiesForHttpClientConnections.desc=Specifies whether to use Java system properties when the JWT Consumer creates HTTP client connections. Set this property to true if you want the connections to use the http* and javax* system properties.
 
+elapsedNBF=Elapsed Not Before Claim
+elapsedNBF.desc=Identifies the elapsed time until JWT will start to be accepted for processing. The value must be a NumericDate.
```
Comment (helyarp) [inline dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties:94]: A nit:  Change "will start" to "starts"

Comment (mmerali96) [reply dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties]: Got it, I updated the description to reflect that change.

**Comment by mmerali96** [comment]
#build

**Comment by mmerali96** [comment]
#build

**Comment by andrea-pichler** [comment]
No L2 message review requested

**Comment by mmerali96** [comment]
#build

**Comment by mmerali96** [comment]
#run-libby-bot

**Comment by mmerali96** [comment]
> The default to -1 is good. However, in runtime, you check elapsedNbfInSeconds > 0, instead of elapsedNbfInSeconds => 0. It is OK to emit nbf to have the same value as 'iat', i.e., elapsedNbfInSeconds ==0

I made the changes to reflect this comment. Thanks!

**Comment by mmerali96** [comment]
#build

**Comment by mmerali96** [comment]
#run-libby-bot

**Pull request merged**